src/openrct2-ui/scripting/ScWidget.hpp

```diff
@@ -300,7 +300,7 @@ namespace OpenRCT2::Scripting
             auto w = GetWindow();
             if (w != nullptr)
             {
-                return WidgetIsDisabled(*w, _widgetIndex);
+                return Ui::WidgetIsDisabled(*w, _widgetIndex);
```

Comment:
Shouldn’t this file also be in the `OpenRCT2::Ui` namespace?

Reply:
Hmm. Not sure. Its scripting which is under OpenRCT2::Scripting. I guess we could make it OpenRCT2::Ui::Scripting but I left it in the current state for that reason.